bindings/java/src/main/java/org/openucx/jucx/ucp/UcpEndpointErrorHandler.java

```diff
@@ -15,5 +15,5 @@
      *             all subsequent operations on this ep will fail with
      *             the error code passed in {@code status}.
      */
-    void onError(UcpEndpoint ep, int status, String errorMsg);
+    void onError(UcpEndpoint ep, int status, String errorMsg) throws Exception;
```

Comment:
maybe throws UcxException ?

Reply:
User may throw it's own exception from this method to catch it later. Like in the example we're throwing: https://github.com/openucx/ucx/pull/5920/files#diff-956587cebcaf92b823b626fdab1f18e9ab40cf426cca63550a387dae285c41afR32 - java ConnectionException to tell that remote part disconnnected.

Reply:
ok